dashi/poolqc/pooling_qc_sample.py

```diff
@@ -112,6 +189,7 @@ def run_URL_update(value):
 )
 def update_lane_options(run_alias):
     run = df[df['Run'] == run_alias]
+    run = run[~run['Run'].isna()]
```

Comment:
I don't understand the purpose of this line. `run = df[df['Run'] == run_alias]` above should already take care of this, as long as `run_alias` is not `NaN`.